roles/test_operator/README.md

```diff
@@ -44,6 +44,7 @@ Execute tests via the [test-operator](https://openstack-k8s-operators.github.io/
 * `cifmw_test_operator_tempest_extra_rpms`: (List) . A list of URLs that point to RPMs that should be installed before the execution of tempest. Note that this parameter has no effect when `cifmw_test_operator_tempest_external_plugin` is used. Default value: `[]`
 * `cifmw_test_operator_tempest_extra_configmaps_mounts`: (List) A list of configmaps that should be mounted into the tempest test pods. Default value: `[]`
 * `cifmw_test_operator_tempest_config`: (Object) Definition of Tempest CRD instance that is passed to the test-operator (see [the test-operator documentation](https://openstack-k8s-operators.github.io/test-operator/crds.html#tempest-custom-resource)). Default value:
+* `cifmw_test_operator_tempest_debug`: (Bool) Run Tempest in debug mode, it keeps the operator pod sleeping infinity (it must only set to `true`only for debugging purposes). Default value: `false`
```

Comment:
```suggestion
* `cifmw_test_operator_tempest_debug`: (Boolean) Run Tempest in debug mode, it keeps the operator pod sleeping infinity (it should only be set to `true` for debugging purposes). Default value: `false`
```

Reply:
Just a nit @rdiazcam I would be happy to approve after this is updated, a suggestion I have would be to add a note/todo regarding @cjeanner 's idea.

Reply:
@lewisdenny, I would create a ticket. Comments tend to get forgotten IMO.